gradle/dependencies.gradle

```diff
@@ -44,6 +48,10 @@ ext.deps = [
         paging: [
             'common': "androidx.paging:paging-common:${versions.paging}",
         ],
+        paging3: [
+            'common': "androidx.paging:paging-runtime:${versions.paging3}",
+            'rx3': "androidx.paging:paging-rxjava3:${versions.paging3}"
```

Comment:
Unused

Reply:
I'll pull rx out of this commit and add it to the next PR. My bad.

Reply:
Ah, didn't realize you had it coming. Can leave it here. Doesn't make a huge difference.